Doc/library/tokenize.rst

```diff
@@ -278,3 +278,15 @@ The exact token type names can be displayed using the :option:`-e` option:
     4,10-4,11:          RPAR           ')'
     4,11-4,12:          NEWLINE        '\n'
     5,0-5,0:            ENDMARKER      ''
+
+Example of tokenizing from another file::
```

Comment:
Changing this sentence to something like "Example of tokenizing a file programmatically::" or "Example of using :func:\`tokenize.tokenize\` programmatically::" would be better.

See for a similar example:

https://github.com/python/cpython/blob/5c403b203510549a3f89d138d3265c5cc0cc12af/Lib/trace.py#L32-L48